src/App.tsx

```diff
@@ -8,7 +8,82 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [inputValue, setInputValue] = useState('');
+  const [selectValue, setSelectValue] = useState('');
+  const [clickedValue, setClickedValue] = useState(0);
+  const [isModal, setIsModal] = useState(false);
+
+  useEffect(() => {
+    getTodos().then(setTodos);
+  }, []);
+
+  let copyTodo = todos;
+
+  const filterList = (search: string) => {
```

Comment:
checkout checklist pt 3 about functions naming conventions and apply it to all functions